programs/survival/potion_master_cleric.sc

```diff
@@ -0,0 +1,138 @@
+// this addon adds potion trades to journyman clerics
+// credit goes to Firigion and the renewable_elytra_trade.sc script
+// credit also goes to _Taser_Monkey_ who provided the nbt data for each custom potion.
+
+// Nbt data for each avalible trade.
+global_potion_trades = [
+	// Dolphins Grace
+	'{
+		sell: {id: "minecraft:%s", Count:1, tag:{
+			CustomPotionEffects:[{Duration:9600, Id: 30, Amplifier: 3}], 
+			CustomPotionColor: 5592575, display: {Name:\'{ "text":"Potion of Grace"}\'}}
+		},
+		buy: {id: "minecraft:emerald", Count: %d},
+		buyB: {id: "minecraft:tropical_fish", Count: 1},
+		uses: 0,
+		demand: -20,
+		priceMutiplier: 0.0,
+		specialPrice: 0,
+		rewardExp: 1,
+		xp: 1
+	}',
+	// Haste
+	'{
+		sell: {id: "minecraft:%s", Count:1, tag:{
+			CustomPotionEffects:[{Duration:9600, Id: 3, Amplifier: 1}], 
+			CustomPotionColor: 16755200, display: {Name:\'{ "text":"Potion of Haste"}\'}}
+		},
+		buy: {id: "minecraft:emerald", Count: %d},
+		buyB: {id: "minecraft:raw_gold", Count: 1},
+		uses: 0,
+		demand: -20,
+		priceMutiplier: 0.0,
+		specialPrice: 0,
+		rewardExp: 1,
+		xp: 1
+	}',
+	// Levitation
+	'{
+		sell: {id: "minecraft:%s", Count:1, tag:{
+			CustomPotionEffects:[{Duration:300, Id: 25, Amplifier: 3}], 
+			CustomPotionColor: 16777215, display: {Name:\'{ "text":"Potion of Levitation"}\'}}
+		},
+		buy: {id: "minecraft:emerald", Count: %d},
+		buyB: {id: "minecraft:phantom_membrane", Count: 1},
+		uses: 0,
+		demand: -20,
+		priceMutiplier: 0.0,
+		specialPrice: 0,
+		rewardExp: 1,
+		xp: 1
+	}',
+	// Wither
+	'{
+		sell: {id: "minecraft:%s", Count:1, tag:{
+			CustomPotionEffects:[{Duration:170, Id: 20, Amplifier: 3}], 
+			CustomPotionColor: 0, display: {Name:\'{ "text":"Potion of Wither"}\'}}
+		},
+		buy: {id: "minecraft:emerald", Count: %d},
+		buyB: {id: "minecraft:wither_rose", Count: 1},
+		uses: 0,
+		demand: -20,
+		priceMutiplier: 0.0,
+		specialPrice: 0,
+		rewardExp: 1,
+		xp: 1
+	}',
+	// Mining Fatigue
+	'{
+		sell: {id: "minecraft:%s", Count:1, tag:{
+			CustomPotionEffects:[{Duration:2800, Id: 4, Amplifier: 0}], 
+			CustomPotionColor: 5592405, display: {Name:\'{ "text":"Potion of Mining Fatigue"}\'}}
+		},
+		buy: {id: "minecraft:emerald", Count: %d},
+		buyB: {id: "minecraft:soul_sand", Count: 1},
+		uses: 0,
+		demand: -20,
+		priceMutiplier: 0.0,
+		specialPrice: 0,
+		rewardExp: 1,
+		xp: 1
+	}',
+	// Blindness
+	'{
+		sell: {id: "minecraft:%s", Count:1, tag:{
+			CustomPotionEffects:[{Duration:330, Id: 15, Amplifier: 0}], 
+			CustomPotionColor: 70000, display: {Name:\'{ "text":"Potion of Blindness"}\'}}
+		},
+		buy: {id: "minecraft:emerald", Count: %d},
+		buyB: {id: "minecraft:ender_eye", Count: 1},
+		uses: 0,
+		demand: -20,
+		priceMutiplier: 0.0,
+		specialPrice: 0,
+		rewardExp: 1,
+		xp: 1
+	}'
+	
+];
+
+global_potion_types = ['splash_potion','lingering_potion','potion'];
+global_potion_emerald_cost = [4,8];
+
+
+_add_trades(cleric, p) -> (
+	// add 0-3 potion trades
+	loop(rand(4),
```

Comment:
If you're gonna add randomly between 0 and 3 trades, should also specify that in readme